libosmscout-gpx/src/osmscout/gpx/Export.cpp

```diff
@@ -479,6 +491,70 @@ bool GpxWritter::WriteWaypoints(const std::vector<Waypoint> &waypoints)
   return true;
 }
 
+bool GpxWritter::WriteExtensions(const Extensions &ext)
```

Comment:
this is not used right? do you plan to use it somehow in future?

Reply:
Yes not necessary because we don't fill simple extensions. To be removed.